infra/build/developer-tools/build/scripts/task_helper_functions.sh

```diff
@@ -307,6 +307,45 @@ function generate_docs() {
     | sort -u)
 }
 
+function check_tflint() {
+  if [[ "${DISABLE_TFLINT:-}" ]]; then
+    echo "DISABLE_TFLINT set. Skipping tflint check."
+    return 0
+  fi
```

Comment:
@apeabody @g-awmalik I was debating btw opt in vs opt out. Considering we would want most modules to onboard to this, I though opt out might be better to accelerate rollout. This does have the drawback that lint could (and likely will) start failing. With opt in, we would need to tackle onboarding repo by repo + clean up flag at some point when all repos have onboarded. Lmk your thoughts.

Reply:
This looks great @bharathkkb! - To my knowledge very few repos are currently tracking CFT `1` or `1.8`, so this shouldn't be immediately effective on the majority repos?  Assuming that is correct, I would make it on by default and we can tackle clean-up at the same time the repo'd CFT version is updated.